extensions/2.0/Vendor/MPEG_audio_spatial/README.md

```diff
@@ -0,0 +1,395 @@
+
+An array of reverb items
+
+* **Type**: [`MPEG_audio_spatial.reverb`](#reference-mpeg_audio_spatial-reverb) `[1-*]`
+* **Required**: No
+
+### MPEG_audio_spatial.extensions
+
+JSON object with extension-specific objects.
+
+* **Type**: `object`
+* **Required**: No
+* **Type of each property**: Extension
+
+### MPEG_audio_spatial.extras
+
+Application-specific data.
+
+* **Type**: [`any`](#reference-any)
+* **Required**: No
+
+
+
+
+---------------------------------------
+<a name="reference-mpeg_audio_spatial-listener"></a>
+## MPEG_audio_spatial.listener
+
+
+
+**`MPEG_audio_spatial.listener` Properties**
+
+|   |Type|Description|Required|
+|---|---|---|---|
+|**id**|`integer`|A unique identifier| &#10003; Yes|
+|**extensions**|`object`|JSON object with extension-specific objects.|No|
+|**extras**|[`any`](#reference-any)|Application-specific data.|No|
+
+Additional properties are allowed.
+
+* **JSON schema**: [MPEG_audio_spatial.listener.schema.json](/Extensions/MPEG_audio_spatial/schema/MPEG_audio_spatial.listener.schema.json)
+
+### MPEG_audio_spatial.listener.id
+
+A unique identifier of the audio listener in the scene.
+
+* **Type**: `integer`
+* **Required**:  &#10003; Yes
+
+### MPEG_audio_spatial.listener.extensions
+
+JSON object with extension-specific objects.
+
+* **Type**: `object`
+* **Required**: No
+* **Type of each property**: Extension
+
+### MPEG_audio_spatial.listener.extras
+
+Application-specific data.
+
+* **Type**: [`any`](#reference-any)
+* **Required**: No
+
+
+
+
+---------------------------------------
+<a name="reference-mpeg_audio_spatial-reverb"></a>
+## MPEG_audio_spatial.reverb
+
+
+
+**`MPEG_audio_spatial.reverb` Properties**
+
+|   |Type|Description|Required|
+|---|---|---|---|
+|**id**|[`any`](#reference-any)|A unique identifier| &#10003; Yes|
+|**bypass**|`boolean`|Indicates if the reverb unit can be bypassed if the audio renderer does not support it.|No, default: `true`|
+|**properties**|[`MPEG_audio_spatial.reverb.property`](#reference-mpeg_audio_spatial-reverb-property) `[1-*]`|An array of property items| &#10003; Yes|
+|**predelay**|`number`|Delay of audio source.|No, default: `0`|
+|**extensions**|`object`|JSON object with extension-specific objects.|No|
+|**extras**|[`any`](#reference-any)|Application-specific data.|No|
+
+Additional properties are allowed.
+
+* **JSON schema**: [MPEG_audio_spatial.reverb.schema.json](/Extensions/MPEG_audio_spatial/schema/MPEG_audio_spatial.reverb.schema.json)
+
+### MPEG_audio_spatial.reverb.id
+
+A unique identifier of the audio reverb unit in the scene.
+
+* **Type**: [`any`](#reference-any)
+* **Required**:  &#10003; Yes
+
+### MPEG_audio_spatial.reverb.bypass
+
+Indicates if the reverb unit can be bypassed if the audio renderer does not support it.
+
+* **Type**: `boolean`
+* **Required**: No, default: `true`
+
+### MPEG_audio_spatial.reverb.properties
+
+Array of items that contains reverbProperties objects describing reverb unit specific parameters
```

Comment:
Since there is no `reverbProperties` symbol, this line should likely say (also in the schema)
> Array of items that contains reverb property objects describing reverb unit specific parameters.

Reply:
Implemented.

Reply:
`MPEG_audio_spatial.reverb.schema.json` schema file was not updated.

Reply:
Thanks @lexaknyazev. This has now been fixed.

Reply:
Updated merge request submitted - thanks